tests/antares/services/local_services/test_area.py

```diff
@@ -424,7 +444,7 @@ def test_st_storage_list_ini_has_correct_default_values(
         assert actual_st_storage_list_ini.parsed_ini.sections() == expected_st_storage_list_ini.sections()
         assert actual_st_storage_list_ini.parsed_ini == expected_st_storage_list_ini
 
-    def test_st_storage_and_ini_have_custom_properties(self, local_study_with_st_storage, actual_st_storage_list_ini):
+    def test_st_storage_and_ini_have_custom_properties(self, local_study_with_renewable):
```

Comment:
Really weird to use _with_renewable in a st_storage test, why did you change this ?

Reply:
Because in this test I test that creating a short term storage with custom properties work, this way it's clear that there is no st storage in the area yet and avoids potential duplicate st storage error.

Reply:
Okay so it would be better to use _w_areas just to make it clearer

Reply:
Corrected